datahub-web-react/src/app/entity/shared/EntityDropdown/CreateGlossaryEntityModal.tsx

```diff
@@ -55,6 +57,7 @@ function CreateGlossaryEntityModal(props: Props) {
         mutation({
             variables: {
                 input: {
+                    id: stagedId?.length ? stagedId : undefined,
```

Comment:
was going to mention that we should ensure it's not a string with only whitespace but `validateCustomUrnId` already does that for us!